### PR TITLE
Releasing TinyMCE 5.7.0 documentation

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-762

Description of Changes:
* Placeholder text

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/plugin-apis/tinymcespellchecker-apis.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{{site.requires_5_7v}}
+
+| Name | Arguments | Description |
+|------| ------| ----------- |
+| addIgnoredWords | words: `string[]`, lang?: `string` | Add an array of words to the `spellchecker_ignore_list` for a specific language. Add the array of words to all languages by omitting the `lang` parameter. |
+| getLanguage | N/A | Get the current language enabled for spellchecking.
+| setLanguage | language: `string` | Set the current language for spellchecking. This must be a supported language code (case-sensitive). For a list of supported languages, see: [Spell Checker Pro - `Supported languages`]({{ site.baseurl }}/plugins/premium/tinymcespellchecker/#supportedlanguages)
```
Comment (tylerkelly13) [inline _includes/plugin-apis/tinymcespellchecker-apis.md:7]:
```suggestion
| setLanguage | language: `string` | Set the current language for spellchecking. This must be a supported language code (case-sensitive). For a list of supported languages, see: [Spell Checker Pro - `Supported languages`]({{ site.baseurl }}/plugins/premium/tinymcespellchecker/#supportedlanguages).
```

File: _includes/events/tinymcespellchecker-events.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+| SpellcheckerIgnore          | `{ word: string }`                           | Fired when a single instance of a word has been marked as ignored.                              |
+| SpellcheckerIgnoreAll       | `{ word: string }`                           | Fired when all instances of a word have been marked as ignored.                                  |
+| SpellcheckError             | `{ message: string }`                        | Fired when a spellchecker error occurs, such as when the Spell Checker Pro service can't be reached. |
+| SpellcheckStart             | N/A                                          | Fired when spellchecking is enabled.                                                            |
+| SpellcheckEnd               | N/A                                          | Fired when spellchecking is disabled.                                                           |
+| SpellcheckerLanguageChanged | `{ language: string, prevLanguage: string }` | Fired when the spellchecking language is changed. {{ site.requires_5_7v }}                      |
```
Comment (tylerkelly13) [inline _includes/events/tinymcespellchecker-events.md:10]:
```suggestion
| SpellcheckerLanguageChanged | `{ language: string, prevLanguage: string }` | Fired when the spellchecking language is changed.<br>{{ site.requires_5_7v }}                      |
```

File: advanced/events.md [hunk 3]
```diff
@@ -132,7 +132,8 @@ The following events are provided by the {{ site.productname }} editor.
...
+| ProgressState          | `{ state: boolean, time?: number }`                                                                                     | Fired when a change to the editor progress state is scheduled using the {{ site.productname }} API [`tinymce.activeEditor.setProgressState()`]({{ site.baseurl }}/api/tinymce/tinymce.editor/#setprogressstate).                                                                                              |
+| AfterProgressState     | `{ state: boolean }`                                                                                                    | Fired after the editor progress state is changed, and the new progress state is in effect. {{ site.requires_5_7v}}                                                         |
```
Comment (tylerkelly13) [inline advanced/events.md:136]:
```suggestion
| AfterProgressState     | `{ state: boolean }`                                                                                                    | Fired after the editor progress state is changed, and the new progress state is in effect.<br>{{ site.requires_5_7v}}                                                         |
```

**Review by ashfordneil — APPROVED** [review]

**Review by lnewson — APPROVED** [review]

File: _includes/plugin-apis/tinymcespellchecker-apis.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+```js
+// ignore words for all languages
+tinymce.activeEditor.plugins.tinymcespellchecker.addIgnoredWords(['tinymce']);
+
+// ignore words for a specific language
+tinymce.activeEditor.plugins.tinymcespellchecker.addIgnoredWords(['TinyMCE', 'tinymce'], 'en_us');
+
+// get the current language used for spellchecking
+tinymce.activeEditor.plugins.tinymcespellchecker.getLanguage()
```
Comment (EkimChau) [inline _includes/plugin-apis/tinymcespellchecker-apis.md:19]: missing semicolon at the end;

File: _includes/plugin-apis/tinymcespellchecker-apis.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+// set the current language to a specific language
+tinymce.activeEditor.plugins.tinymcespellchecker.setLanguage('sv')  // e.g. Swedish
```
Comment (EkimChau) [inline _includes/plugin-apis/tinymcespellchecker-apis.md:22]: missing semicolon at the end;

**Review by EkimChau — APPROVED** [review]

File: _includes/plugin-apis/tinymcespellchecker-apis.md (hunk 5 above)
Comment (tylerkelly13) [reply _includes/plugin-apis/tinymcespellchecker-apis.md:22]:
```suggestion
tinymce.activeEditor.plugins.tinymcespellchecker.setLanguage('sv');  // e.g. Swedish
```

File: _includes/plugin-apis/tinymcespellchecker-apis.md (hunk 4 above)
Comment (tylerkelly13) [reply _includes/plugin-apis/tinymcespellchecker-apis.md:19]:
```suggestion
tinymce.activeEditor.plugins.tinymcespellchecker.getLanguage();
```

**Review by JamesToohey — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**